modules/event-list/types.ts

```diff
@@ -1,4 +1,22 @@
+import type {NavigationScreenProp} from 'react-navigation'
+import type {EventType} from '@frogpond/event-type'
+
 export type PoweredBy = {
 	title: string
 	href: string
 }
+
+type Navigation = NavigationScreenProp<{
+	params: {event: EventType; poweredBy?: PoweredBy}
+}>
+
+export type Props = {
+	navigation: Navigation
+}
+
+export type NavigationHeaderProps = {
+	title: string
+	headerRight: JSX.Element
+}
+
+export type NullableElement = JSX.Element | null
```

Comment:
My only comment is whether this typealias makes sense. It's short enough that I'm torn about just inlining it everywhere?

Reply:
I've just added another one below this... If we do this often enough, are we in favor of adding a set of convenience types, or do we prefer them inlined everywhere? I understand the latter approach would involve an import to use it...

Reply:
Ultimately I'd prefer to have a "named" type for everything to allow for more sharing and readability but I'm open to both approaches.

Reply:
I agree that `T | null` could probably best be handwritten. That makes it clear that it's a one-or-the-other choice rather than a potentially more complex set of options. This way, if you're just reading the code without any extra tools to inline types, you could easily see the two possibilities.

If we wanted to use a type to encode `| null`, I would rather have a generic.

```ts
export type Nullable<T> = T | null
```

Reply:
Okay! I've walked these back.